core/templates/dev/head/css/oppia.css

```diff
@@ -119,6 +119,78 @@ body {
   margin-top: 15px;
 }
 
+.oppia-profile-tabs md-content {
+  background-color: white !important; 
+}
+
+.oppia-profile-tabs md-content md-tabs {
+  background: #f6f6f6;
+  border: 1px solid #e1e1e1; 
+}
+
+.oppia-profile-tabs md-content md-tabs md-tabs-wrapper {
+  background: white; 
+}
+
+.oppia-profile-tabs md-content h1:first-child {
+  margin-top: 0; 
+}
+
+.oppia-profile-tabs md-tab {
+  background-color: #009688;
+}
+
+.oppia-profile-tabs md-tab-ink-bar {
+  background-color: black;
+}
+
+md-tabs. md-padding {
+  width: 20%;
+}
+
+.oppia-profile-tabs md-tab-label {
+    color: #ffffff;
+}
+
+.stat-container-top {
+    display: inline-block;
+    width: 45%;
+}
+
+.stat-container {
+    display: inline-block;
+    width: 25%;
+}
+
+.stat-big-container {
+    display: block;
```

Comment:
Style nits: throughout this file, use 2-space indentation, and alphabetize the CSS attrs (so, for this rule, it would be display, then margin-bottom, then text-align, then width). This makes it easier to scan for a particular attribute.